image-swipe-common.ts

```diff
@@ -89,8 +89,6 @@ export const itemsProperty = new Property<ImageSwipeBase, any[] | ItemsSource>({
         if (newValue instanceof ObservableArray) {
             addWeakEventListener(newValue, ObservableArray.changeEvent, target.refresh, target);
         }
-
-        target.refresh();
```

Comment:
Dont think removing this is correct. Calling `refresh()` from my previous testing is obligatory and needed in order on change of the items the views to be updated correctly. Even the core `list-view` plugins has this: https://github.com/NativeScript/NativeScript/blob/5f22594eb78f4950943766f280901e38ae9de5ab/tns-core-modules/ui/list-view/list-view-common.ts#L181 

In order to handle the error that this is supposed to fix, it is better to check for iOS if the view is init here:
https://github.com/PeterStaev/nativescript-image-swipe/blob/3f0336520699f6ad03e3c69e6c0578868b200742/image-swipe.ios.ts#L302

There is already a similar check for android here:
https://github.com/PeterStaev/nativescript-image-swipe/blob/3f0336520699f6ad03e3c69e6c0578868b200742/image-swipe.android.ts#L78-L80

Reply:
I was initially thinking the same. But wasn't able to reproduce a scenario where this removal was an issue. I will rework this to make a check in the platform if the `nativeViewProtected` is present, this was my first solution to this issue.